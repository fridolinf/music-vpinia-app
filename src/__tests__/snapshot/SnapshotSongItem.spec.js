import { RouterLinkStub, shallowMount } from '@vue/test-utils'
import SongItem from '@/components/SongItem.vue'
import { describe, expect, test } from 'vitest'
describe('SongItem snapshot', () => {
  test('Render correctly', () => {
    const song = {}
    const wrapper = shallowMount(SongItem, {
      props: { song },
      global: {
        components: {
          'router-link': RouterLinkStub
        }
      }
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
