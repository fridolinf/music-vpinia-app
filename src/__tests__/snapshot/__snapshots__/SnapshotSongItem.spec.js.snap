// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SongItem snapshot > Render correctly 1`] = `
<li
  class="flex justify-between items-center p-3 pl-6 cursor-pointer transition duration-300 example border-gray-400 border-b song-id-undefined"
  id="song-id-undefined"
>
  <div>
    <router-link-stub
      class="font-bold block text-black song-item"
      custom="false"
      to="[object Object]"
    />
    <span
      class="text-gray-500 text-sm"
    />
  </div>
  <div
    class="text-gray-600 text-lg"
  >
    <router-link-stub
      custom="true"
      to="[object Object]"
    />
  </div>
</li>
`;
